python/pyproject.toml

```diff
@@ -35,13 +35,42 @@ markers = [
   "no_bad_cuml_array_check: Test that should not check for bad CumlArray uses",
 ]
 
-testpaths = "cuml/tests"
+testpaths = [
+  "cuml/tests",
+  "cuml/tests/dask",
+  "cuml/tests/experimental",
+  "cuml/tests/explainer",
+  "cuml/tests/stemmer_tests",
+  "cuml/tests",
+]
 
 filterwarnings = [
-  "error::FutureWarning:cuml[.*]", # Catch uses of deprecated positional args in testing
+  "error::FutureWarning",
+  "error::DeprecationWarning",
   "error:::cudf",
   "ignore:[^.]*ABCs[^.]*:DeprecationWarning:patsy[.*]",
   "ignore:(.*)alias(.*):DeprecationWarning:hdbscan[.*]",
+  # TODO
```

Comment:
Can we open a GH issue with the content of this file there? Will be great to track tackling these.

Reply:
Good idea. Opened and linked https://github.com/rapidsai/cuml/issues/5879 https://github.com/rapidsai/cuml/issues/5878